ibis/backends/clickhouse/tests/test_client.py

```diff
@@ -135,18 +133,18 @@ def test_table_info(alltypes):
     assert buf.getvalue() is not None
 
 
-def test_execute_exprs_no_table_ref(con):
-    cases = [(L(1) + L(2), 3)]
+# def test_execute_exprs_no_table_ref(con):
```

Comment:
Same as below I think we can use a `list` here and keep the test as is otherwise.

Reply:
My plan was the same, but requires more work on the SQL backends since expr would be a list [here](https://github.com/ibis-project/ibis/blob/master/ibis/backends/base/sql/compiler/query_builder.py#L561-L582).